### DISCUSSION TITLE
Quiz21 Q4

### DISCUSSION THREAD

**Discussion by ln2-muyuan** [AmazingAng/WTF-Solidity#410] [Q&A] [answered]
想问问第三个call_set(20)为什么会报错吖
![image](https://user-images.githubusercontent.com/73374854/216758467-3b3b1755-8675-4ca6-8ddc-328eacc6c689.png)

**Comment by LBJyang** [comment]
我好奇你们是怎么找到正确答案的

**Reply by ln2-muyuan** [reply]
运行代码试出来的，，但不知道理由是啥

**Reply by luyao-pilgrim** [reply]
我试了下没有错，都是可以运行的

**Comment by kazimics** [comment, marked as answer]
不会报错，所以3是错误的，不选啊
